third_party/move/move-vm/runtime/src/storage/loader/eager.rs

```diff
@@ -67,4 +70,34 @@ where
     }
 }
 
+impl<'a, T> NativeModuleLoader for EagerLoader<'a, T>
+where
+    T: ModuleStorage,
+{
+    fn charge_native_result_load_module(
+        &self,
+        gas_meter: &mut impl GasMeter,
+        traversal_context: &mut TraversalContext,
+        module_id: &ModuleId,
+    ) -> PartialVMResult<()> {
+        let arena_id = traversal_context
+            .referenced_module_ids
+            .alloc(module_id.clone());
+        check_dependencies_and_charge_gas(self.module_storage, gas_meter, traversal_context, [(
+            arena_id.address(),
+            arena_id.name(),
+        )])
+        .map_err(|err| {
+            err.to_partial().append_message_with_separator(
+                '.',
+                format!(
+                    "Failed to charge transitive dependency for {}. Does this module exists?",
```

Comment:
typo: "exist?"

Reply:
Copy pasted old code, good catch, I think it's ok to change the message as it's discarded anyway and it seems like invariant violation in any case